### ISSUE TITLE
闻仲模型提示attention mask和pad token id问题，输出出现乱码

### ISSUE THREAD

**Issue by puluwen** [IDEA-CCNL/Fengshenbang-LM#319]
>>> generator("已知情感极性列表：['正面', '负面', '中性']，那么句子“这个电脑性能非常棒”的情感极性是", max_length=100, num_return_sequences=1)
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.
[{'generated_text': "已知情感极性列表：['正面', '负面', '中性']，那么句子“这个电脑性能非常棒”的情感极性是：“这个电�"}]

**Comment by Zimix0** [comment]
感谢使用闻仲模型，我测试了一下你的例子：
<img width="1248" alt="image" src="https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/93325445/29801418-34db-4b3a-90cb-be062e8f25dc">
generator方法里面封装的方法可以设置一下相应的参数，就可以正常运行。
关于attention mask以及pad token id也是，设置即可，不设置会使用默认值。
具体代码可以参考这里：https://huggingface.co/IDEA-CCNL/Wenzhong-GPT2-110M
tokenizer的对象是有attention mask的：
<img width="1154" alt="image" src="https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/93325445/cefee73b-ffb1-474b-968c-15165f04b7ff">
还有“�”并不是输出的乱码，模型的分词采用的是bpe算法，中文的字或者词大概率会变成多个token，我们设置了最大生成长度，容易导致最后的字或者词没有生产完整，最后就解码了一个字的部分token所有有这种情况的出现。
最后，这是gpt结构的模型，不是如今如火如荼的chatgpt，可能直接这样推理的效果还是有更大的随机性。可以看一下我们最新的ziya模型：https://huggingface.co/IDEA-CCNL/Ziya-LLaMA-13B-v1
你的输入可能这样修改了一下后效果会更满足你的期待(不过还是有一定的随机性，并没有那么聪明)：
<img width="1258" alt="image" src="https://github.com/IDEA-CCNL/Fengshenbang-LM/assets/93325445/cae25cc3-1b0c-4ce7-a9ea-cc1fa5dc2651">
